starfish/pipeline/features/spots/decoder/iss.py

```diff
@@ -1,6 +1,4 @@
 from ._base import DecoderAlgorithmBase
```

Comment:
I adjusted any tools used by the ISS pipeline to use `Codebook` + `IntensityTable`. Because the decoders are generalized and associated with the `Codebook`, this module is substantially simplified.